src/features/foods/components/Form/AddDrawerContent/index.tsx

```diff
@@ -76,8 +78,8 @@ export const AddDrawerContent = ({
       alert('Something went wrong. Please try again.');
     } else {
       alert('Successfully created');
-      form.reset();
```

Comment:
`form.reset()` was not necessary because it anyway reset the form values on `router.refresh()`